### ISSUE TITLE
P3V3 rail is programmed for 3.44V

### ISSUE THREAD

**Issue by ryan-summers** [sinara-hw/Booster#367]
The P3V3 rail is actually ~3.44V as opposed to the expected 3.3V level.

This plays a key role when translating analog samples into real-world voltages. The references in the schematic should be updated to indicate the real voltage of the rail or the resistor divider should be updated to correctly output 3.3V

**Comment by hartytp** [comment]
@ryan-summers thanks for all the good work on this.

Reminding myself, the 3V3 rail is currently supplied by the SMPS IC12
![image](https://user-images.githubusercontent.com/21218399/90630883-bd86a880-e219-11ea-8841-cd0a51624892.png)

With the current resistor values (chosen to be E24 I assume) and a nominal Vref of 0.8V, we expect 3.434 which is consistent with your observations.

:+1: to adding a note on the schematic.

**Comment by hartytp** [comment]
To calibrate expectations here, the critical functionality for this ADC is measuring the PA FET current. The [InAmp](https://www.analog.com/media/en/technical-documentation/data-sheets/LT6106.pdf) offset voltage is 150uV typical/350uV max.

50mA*0.1Ohm gives 5mVsense at the InAmp. That's a 3% accuracy (ignoring any other source of error).

Having an ADC Vref of 3.4 is a 4% inaccuracy. So, it doesn't make a significant difference to the overall current sense accuracy (in any case, biasing the FETs to a constant gate voltage is a slightly crude way of doing things and I'd expect the variations in FET current due to temperature to swamp the inaccuracies in the initial setting).

tl;dr I think this is worth an annotation, but not a change to the PCB

**Comment by ryan-summers** [comment]
> tl;dr I think this is worth an annotation, but not a change to the PCB
Agreed.

I just want to point out I'm not trying to use this as a precision measurement device, I'm just trying to show the most accurate information possible to the user
